src/etc/dec2flt_table.py

```diff
@@ -97,7 +96,7 @@ def print_proper_powers(min_exp, max_exp, bias):
     print('#[rustfmt::skip]')
     typ = '[(u64, u64); N_POWERS_OF_FIVE]'
     print('pub static POWER_OF_FIVE_128: {} = ['.format(typ))
-    lo_mask = (1 << 64) - 1
+    # lo_mask = (1 << 64) - 1
```

Comment:
Let's just delete this line since it seems to be unused and I don't think is particularly interesting content-wise.

Reply:
Agreed, it's gone now